src/templates/core/proposals/proposal_editorial_review.html

```diff
@@ -10,23 +10,50 @@
 
   <table class="table table-bordered">
     <tr>
-      <th>ID</th>
+      <th>Name</th>
       <th>Assigned</th>
       <th>Status</th>
+      <th>Recommendation</th>
       <th></th>
     </tr>
   {% for review in editorial_review_assignments %}
     <tr>
-      <td>{{review.pk}}</td>
-      <td>{{review.assigned}}</td>
-      <td width="25%">
+      <td>
+          <a href="{% url 'view_profile_readonly' review.user.id %}">
+              {{ review.user.first_name }}
+              {% if review.user.middle_name %}
+                {{ review.user.middle_name }}
+              {% endif %}
+              {{ review.user.last_name }}
+          </a>
+          <sup>
+              <a href="core/email.html"
+                 onclick="return popitup('/email/proposal/{{ proposal.id }}/user/{{ review.user.id }}/',
+                 'proposal_{{ review.user.id }}')"
+                 class="fa fa-envelope-o btn btn-xs btn-default">
```

Comment:
I assume this is supposed to be an empty anchor tag?

Reply:
It's an email icon that links to a separate email page